config/v1/types_project.go

```diff
@@ -0,0 +1,36 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Project holds cluster-wide information about Project.  The canonical name is `cluster`
+type Project struct {
```

Comment:
Would probably say this becomes `SelfService` but we don't have to create it yet.  Need to think about it.